src/pluto/pluto.py

```diff
@@ -418,9 +418,8 @@ def generate_graph(r, total, title):
             st.write(df)
             st.info(
                 """
-            The mismatch graphs do not include lots that formerly had a value and are now null, or vice versa.
-            These differences are captured in the null graph, which shows the percent change in lots with a null value.
-            Hovering over a point shows you the number of null records in the more recent file. The number of such changes should be small.
+                The above graph highlights records that formerly had a value and are now NULL, or vice versa.
+                The number records going from NULL to not NULL or vice versa should be small for any field.
```

Comment:
I think it makes sense for this to be right below the graph it is referring to as supposed to the table. This would follow the convention established in the other graphs and subsequent tables

Reply:
Just move `st.info` right after the `st.plotly_chart` on line 417